crates/optimism/node/src/lib.rs

```diff
@@ -22,6 +22,10 @@ pub use node::OpNode;
 
 pub mod txpool;
 
+/// Helpers for running test node instances.
```

Comment:
Moved because we'd like to use it in both `p2p` and `it` tests. Can also move my test to p2p & revert the move if you prefer